conf/locations-and-venues.yaml

```diff
@@ -1,10 +1,13 @@
 London:
-  - London FSAC:
+  - LONDON_FSAC:
       description: Bush House
```

Comment:
[note]
It's not really description.. it's display value (or label) that we will show on UI. We can't show LONDON_FSAC.
I suggest to change it in scope of my story FSET-1553 if you're ok with it.